buildSrc/src/main/groovy/com/rocana/gradle/avro.groovy

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2016 Rocana.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rocana.gradle
```

Comment:
For now, yes. The plan was (is?) is extract the content of buildSrc into a separate module. I just left it in here to simplify the review process. I have a local branch that removes the content of buildSrc to be done after I push the avro plugin as a separate project.